cg/cli/workflow/commands.py

```diff
@@ -0,0 +1,188 @@
+    """
+    Store finished analysis files in Housekeeper
+    """
+    analysis_api: AnalysisAPI = context.obj["analysis_api"]
+    analysis_api.verify_case_id_in_statusdb(case_id=case_id)
+    if dry_run:
+        LOG.info("Dry run: Would have stored deliverables for %s", case_id)
+        return
+    try:
+        analysis_api.upload_bundle_housekeeper(case_id=case_id)
+        analysis_api.upload_bundle_statusdb(case_id=case_id)
+        analysis_api.set_statusdb_action(case_id=case_id, action=None)
+    except Exception as exception_object:
+        analysis_api.housekeeper_api.rollback()
+        analysis_api.status_db.rollback()
+        LOG.error("Error storing deliverables for case %s - %s", case_id, exception_object)
+        raise
+
+
+@click.command("store-available")
+@OPTION_DRY
+@click.pass_context
+def store_available(context: click.Context, dry_run: bool) -> None:
+    """
+    Store bundles for all finished analyses in Housekeeper
+    """
+    analysis_api: AnalysisAPI = context.obj["analysis_api"]
+    exit_code: int = EXIT_SUCCESS
+    for case_obj in analysis_api.get_cases_to_store():
+        LOG.info("Storing deliverables for %s", case_obj.internal_id)
+        try:
+            context.invoke(store, case_id=case_obj.internal_id, dry_run=dry_run)
+        except Exception as exception_object:
+            LOG.error("Error storing %s: %s", case_obj.internal_id, exception_object)
+            exit_code = EXIT_FAIL
+    if exit_code:
+        raise click.Abort
+
+
+@click.command("clean-run-dir")
+@click.option("-y", "--yes", is_flag=True, help="Skip confirmation")
+@click.option("-d", "--dry-run", is_flag=True, help="Shows cases and files that would be cleaned")
+@ARGUMENT_CASE_ID
+@click.pass_context
+def clean_run_dir(context, yes: bool, case_id: str, dry_run: bool = False):
+    """Remove Balsamic run directory"""
+
+    analysis_api = context.obj["analysis_api"]
+    analysis_api.verify_case_id_in_statusdb(case_id)
+    analysis_api.verify_case_path_exists(case_id=case_id)
+    analysis_api.check_analysis_ongoing(case_id=case_id)
+    analysis_path: Path = analysis_api.get_case_path(case_id)
+
+    if dry_run:
+        LOG.info(f"Would have deleted: {analysis_path}")
+        return EXIT_SUCCESS
+
+    if yes or click.confirm(f"Are you sure you want to remove all files in {analysis_path}?"):
+        if analysis_path.is_symlink():
+            LOG.warning(
+                f"Will not automatically delete symlink: {analysis_path}, delete it manually",
+            )
+            return EXIT_FAIL
+
+        shutil.rmtree(analysis_path, ignore_errors=True)
+        LOG.info("Cleaned %s", analysis_path)
+        analyses: list = analysis_api.status_db.family(case_id).analyses
+        if analyses:
+            for analysis_obj in analyses:
+                analysis_obj.cleaned_at = analysis_obj.cleaned_at or dt.datetime.now()
+                analysis_api.status_db.commit()
```

Comment:
```suggestion
        for analysis_obj in analyses:
            analysis_obj.cleaned_at = analysis_obj.cleaned_at or dt.datetime.now()
            analysis_api.status_db.commit()
```
analyses is a list so no problem to loop over an empty list